src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.svg'
```

Comment:
This is necessary to get past an import error on the loading indicator on line 2 of `Spinner.tsx`